src/service/json-schema-service.js

```diff
@@ -1,37 +1,48 @@
-import publishSchema from '../controllers/http-api/request-schema/publish-schema.js';
-import updateSchema from '../controllers/http-api/request-schema/update-schema.js';
-import getSchema from '../controllers/http-api/request-schema/get-schema.js';
-import querySchema from '../controllers/http-api/request-schema/query-schema.js';
-import bidSuggestionSchema from '../controllers/http-api/request-schema/bid-suggestion-schema.js';
-import localStoreSchema from '../controllers/http-api/request-schema/local-store-schema.js';
+import path from 'path';
+import appRootPath from 'app-root-path';
 
 class JsonSchemaService {
     constructor(ctx) {
         this.blockchainModuleManager = ctx.blockchainModuleManager;
     }
 
-    bidSuggestionSchema() {
-        return bidSuggestionSchema(this.blockchainModuleManager.getImplementationNames());
+    async loadSchema(version, schemaName) {
+        const schemaPath = path.resolve(
+            appRootPath.path,
+            `src/controllers/http-api/${version}/request-schema/${schemaName}-schema-${version}.js`,
+        );
+        const schemaModule = await import(schemaPath);
+        const schemaFunction = schemaModule.default;
+
+        if (schemaFunction.length >= 1) {
+            return schemaFunction(this.blockchainModuleManager.getImplementationNames());
```

Comment:
the issue with this is that all schema functions will have to have to same arguments. Ideally we would want to have the ability to have schemas with different parameters